mariner/mariner.go

```diff
@@ -154,8 +152,24 @@ func (c Config) update(version, path string) error {
 
 	return nil
 }
+func (c Config) saveAdvisoryPerYear(dirName string, def Definition) error {
+	// Mariner uses `<ID>_<last_number_from_version>` format for `advisory_id`.
```

Comment:
AFAIK, they use hyphens.

```suggestion
	// Mariner uses `<ID>-<last_number_from_version>` format for `advisory_id`.
```

Reply:
We should start calling Azure Linux.

```suggestion
	// Azure Linux uses `<ID>_<last_number_from_version>` format for `advisory_id`.
```

Reply:
fixed in https://github.com/aquasecurity/vuln-list-update/pull/271/commits/995ea97f301c04bacd37987086c6d8992add1703